config/common/errors.go

```diff
@@ -69,6 +69,7 @@ var (
 	ErrUserFieldSupport       = errors.New("fields other than \"name\" and \"ssh_authorized_keys\" are not supported in this spec version")
```

Comment:
This commit should also be `fixup! config: add initial flatcar variant`, or else `config: add initial flatcar variant` won't compile after the fixups are applied.

(FYI, we typically use a force-push PR workflow where we modify existing commits between review rounds, rather than pushing explicit fixup commits.)

Reply:
Ok, the idea was to move the commit: `config/error: add error for non-supported clevis` as initial commit during the rebase. Thanks for sharing your workflow, will rebase/force-push for the next batch.

Reply:
Ah, yeah, that makes sense.